src/app/member/components/MemberActionRender.tsx

```diff
@@ -1,14 +1,21 @@
-import ActionsMenu from '@/components/common/ActionsMenu';
+import ActionsMenu from '@/components/common/ActionMenu/ActionsMenu';
 import { Info, Trash2, Edit, Send } from "lucide-react";
 import React from 'react';
 
-// When we will implement in table we will update this type
-type Props ={
+interface MemberData {
+  id: number;
+  name: string;
+  phone: string;
+  // shift: string;
   role: string;
   status: string;
 }
+type Props ={
+  memberData: MemberData;
```

Comment:
I'm not in favor of using the keyword `Data` in any case.

Reply:
Fixed